File: Groove.SP/groove-sp-web/src/app/features/compliance/models/cargo-loadability.model.ts
import { EquipmentType } from 'src/app/core';

export class CargoLoadabilityModel {
    name: string;
    equipmentType: EquipmentType;
    cyMinimumCBM: number;
    cyMaximumCBM: number;
    cfsMinimumCBM: number | null;
    cfsMaximumCBM: number | null;
}

File: Groove.SP/Groove.SP.Core/Entities/CargoLoadabilityModel.cs
using Groove.SP.Core.Models;

namespace Groove.SP.Core.Entities
{
    public class CargoLoadabilityModel : Entity
    {
        public long Id { get; set; }

        public long BuyerComplianceId { get; set; }

        public BuyerComplianceModel BuyerCompliance { get; set; }

        public string Name { get; set; }

        public EquipmentType EquipmentType { get; set; }

        public decimal CyMinimumCBM { get; set; }

        public decimal CyMaximumCBM { get; set; }

        public decimal CfsMinimumCBM { get; set; }

        public decimal CfsMaximumCBM { get; set; }
    }
}